CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to COS Alerter
+
+[Bugs](https://github.com/canonical/cos-alerter/issues)
+
+[Pull Requests](https://github.com/canonical/cos-alerter/pulls)
+
+## Setup
+
+* `python3 -m venv venv`
+* `source venv/bin/activate`
+
+## Run
+
+* `pip install .`
+* `cos-alerter`
+
+## Run Tests
+
+* `pip install tox`
+* `tox`
+
+## Run With Docker
```

Comment:
Move this section next to or under "Run"?

Reply:
Done.